src/components/page/blocks/ContentBlock.jsx

```diff
@@ -10,6 +10,7 @@ import {
   gridCopyMiddle,
   gridCopyBottom,
 } from '../page.module.scss';
+import i18nComponents from '../../../lib/i18nComponents';
```

Comment:
pulled the translation components out of here and generalized them since image captions often have links inside